### PR TITLE
executor: expose `chunk.Column` to prepare for vectorized expression evaluation

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve? 
Expose `chunk.Column` and some methods of it to let `Expression` can access data directly by it instead of using `Row` and `RowIter`, which is prepared for vectorized expression evaluation.

### What is changed and how it works?
Just expose `chunk.Column` and some methods of it.
**No logical changes.**

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: util/chunk/column.go [hunk 1]
```diff
@@ -130,49 +137,56 @@ func (c *column) appendNull() {
...
 
-func (c *column) finishAppendFixed() {
+func (c *Column) finishAppendFixed() {
 	c.data = append(c.data, c.elemBuf...)
 	c.appendNullBitmap(true)
 	c.length++
 }
 
-func (c *column) appendInt64(i int64) {
+// AppendInt64 appends a int64 value into this Column.
```
Comment (SunRunAway) [inline util/chunk/column.go]: an int64

File: util/chunk/column.go [hunk 2]
```diff
@@ -130,49 +137,56 @@ func (c *column) appendNull() {
...
+func (c *Column) AppendInt64(i int64) {
 	*(*int64)(unsafe.Pointer(&c.elemBuf[0])) = i
 	c.finishAppendFixed()
 }
 
-func (c *column) appendUint64(u uint64) {
+// AppendUint64 appends a uint64 value into this Column.
```
Comment (SunRunAway) [inline util/chunk/column.go]: an uint64

Comment (coocood) [reply util/chunk/column.go]: Should be `a uint64`

Comment (SunRunAway) [reply util/chunk/column.go]: You're right, sorry to @qw4990

Comment (qw4990) [reply util/chunk/column.go]: hmm, fixed.

File: util/chunk/column.go (hunk 1 above)
Comment (qw4990) [reply util/chunk/column.go]: fixed.

**Pull request merged**